DnD6/Scenes/Intro/Intro.gd

```diff
@@ -0,0 +1,37 @@
+extends Control
+
+var animationFinished = false
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	$MarginContainer/AnimationPlayer.play("TextDisplay")
+	set_process(true)
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(_delta):
+	if $MarginContainer/AnimationPlayer.is_playing() == false and animationFinished == false:
+		onAnimationFinished()
```

Comment:
use signals instead of checking every frame

Reply:
AnimationPlayer does not appear to have a stop signal which signal should be used here?

Reply:
![image](https://github.com/Tadd12/DnD6/assets/95188667/6bf99f69-76a9-4f61-8933-775742a59e3f)

Reply:
Added that, thanks

Reply:
it currently gives me an error
ERROR: Error calling from signal 'animation_finished' to callable: 'Control(Intro.gd)::onAnimationFinished': Method expected 0 arguments, but called with 1.
   at: emit_signalp (core/object/object.cpp:1140)

Reply:
Weird it works for @FFranziskus13 an me

Reply:
Changed the func so now it should work